content/jmccann/clair/index.md

```diff
@@ -0,0 +1,88 @@
+---
+date: 2017-02-09T00:00:00+00:00
+title: Clair
+author: jmccann
+tags: [ docker, security ]
+repo: jmccann/drone-clair
+logo: clair.svg
+image: jmccann/drone-clair
+---
+
+The Clair plugin submits your docker image to your [Clair](https://github.com/coreos/clair)
+server to scan your docker image for security vulnerabilities.
+
+The below pipeline configuration demonstrates simple usage:
+
+```yaml
+pipeline:
+  clair:
+    image: jmccann/drone-clair:1
+    url: http://clair.company.com
+    username: johndoe
+    password: mysecret
+    scan_image: python:2.7
+```
+
+To verify https/ssl connections with a different CA certificate use `ca_cert`
+
+```diff
+pipeline:
+  clair:
+    image: jmccann/drone-clair:1
+    url: http://clair.company.com
+    username: johndoe
+    password: mysecret
+    scan_image: python:2.7
++   ca_cert: |
++     -----BEGIN CERTIFICATE-----
++     MII...
++     -----END CERTIFICATE-----
+```
+
+# Secrets
+
```

Comment:
I updated the section to show an example using interpolation.  Since plugins are usually a users first experience for dealing with secrets I think it's still valuable to show a quick example with a link to docs for more information.